core/src/test/scala/io/snappydata/CommandLineToolsSuite.scala

```diff
@@ -24,6 +24,7 @@ import org.apache.commons.io.output.TeeOutputStream
 import org.apache.spark.sql.collection.Utils
 
 import scala.sys.process._
+import io.snappydata.SnappyTestRunner
```

Comment:
Is this required? It's the same package.

Reply:
yes, that's redundant. Thanks